utils/cosmos/iscn/sign.ts

```diff
@@ -37,14 +40,30 @@ export function formatISCNTxPayload(payload: ISCNRegisterPayload): ISCNSignPaylo
     for (let i = 0; i < authorNames.length; i += 1) {
       const authorName = authorNames[i]
       const authorUrl = authorUrls[i]
-      const authorId = authorWallets[i]
+      let authorId = ''
+      authorWallets[i].forEach((a: any) => {
+        if(a.type === 'cosmos'){
+          authorId =`did:cosmos:${a.address.slice(6)}`
+        }else if (a.type ==='eth' && !authorId ){
+          authorId = `did:eth:${a.address}`
+        }else{
+          return ''
+        }
+        return undefined
+      })
+      const authorWalletAddress = authorWallets[i]
+      const likerId = likerIds[i]
+      const authorDescription = descriptions[i]
       const isNonEmpty = authorUrl || authorName || authorId
       if (isNonEmpty) {
         stakeholders.push({
             entity: {
-              '@id': authorId ? `did:cosmos:${authorId.slice(6)}` : authorUrl || authorName,
-              name: authorName,
-              url: authorUrl,
+              '@id': authorId || authorUrl || authorName,
+                name: authorName,
+                url: authorUrl,
+                authorWalletAddress,
+                likerId,
```

Comment:
LikerId should be written as a `url` to `https://like.co/{likerId}` instead of a field called `likerId`